scripts/js/lib/api/processHtml.ts

```diff
@@ -157,6 +158,13 @@ export function removeDownloadSourceCode($main: Cheerio<any>): void {
   $main.find("p > a.reference.download.internal").closest("p").remove();
 }
 
+function removePublicMembersRubric($main: Cheerio<any>): void {
```

Comment:
Can you please add docstring explaining why we do this? Maybe mention the better solution is inlining headers and link to that issue.

Reply:
Good idea, thanks